tools/serverpod_cli/test/test_variants.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:meta/meta.dart';
```

Comment:
Not sure where to put this file, so for now it is at the root of the `serverpod_cli` test directory.